src/textures.ts

```diff
@@ -0,0 +1,73 @@
+import r from "./bindings/raylib"
+import { Color } from "./types/colors"
+import { Vector2, Vector4, Rectangle, NPatchInfo } from "./types/maths"
+import { Image, Texture2D, TextureCubemap, Font, RenderTexture2D } from "./types/texture"
+
+export const LoadImage: (fileName: string) => Image = r.LoadImage //Load image from file into CPU memory (RAM) 
+export const LoadImageEx: (pixels: Color, width: number, height: number) => Image = r.LoadImageEx //Load image from Color array data (RGBA - 32bit) 
+export const LoadImagePro: (data: any, width: number, height: number, format: number) => Image = r.LoadImagePro //Load image from raw data with parameters 
+export const LoadImageRaw: (fileName: string, width: number, height: number, format: number, headerSize: number) => Image = r.LoadImageRaw //Load image from RAW file data 
+export const ExportImage: (image: Image, fileName: string) => void = r.ExportImage //Export image data to file 
+export const ExportImageAsCode: (image: Image, fileName: string) => void = r.ExportImageAsCode //Export image as code file defining an array of bytes 
```

Comment:
All of these exports weren't needed before, as they were exported from the Node addon. Is there a way to auto export them so that we don't have to redeclare them?

Reply:
To my knowledge it is not possible to apply a type declaration to a function post creation, so this was the solution that I went with, it is very possible that I just missed something however

Reply:
Hmmm, that may duplicate some work later on when adding new raylib functions. Hopefully we figure out a way around that.